lexers/troff.lua

```diff
@@ -0,0 +1,30 @@
+-- Copyright 2015-2017 David B. Lamkins <david@lamkins.net>. See LICENSE.
+-- man/roff LPeg lexer.
+
+local lexer = lexer
+local P, R, S = lpeg.P, lpeg.R, lpeg.S
+
+local lex = lexer.new(...)
+
+-- Registers and groff's structured programming.
+lex:add_rule('keywords', lex:tag(lexer.KEYWORD, (lexer.starts_line('.') * lexer.space^0 * (P('while') + P('break') + P('continue') + P('nr') + P('rr') + P('rnn') + P('aln') + P('\\}'))) + P('\\{'))
+
+-- Markup.
+lex:add_rule('escape_sequences', lex:tag(lexer.VARIABLE,
+	P('\\') * ((P('s') * S('+-')) + S('*fgmnYV') + P("")) *
```

Comment:
The `P("")` is not needed.

Reply:
It is needed because sequences such as `\(em` or `\[u00E0]` need to be highlighted. But I think it is cleaner when implemented as `^-1` around ((P('s') * S('+-')) + S('*fgmnYV'))`